### PR TITLE
react-native - Added nativeConfig to WebViewProps

### PR DESCRIPTION
**Pull request by TheAlmightyBob**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://facebook.github.io/react-native/docs/webview#nativeconfig
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/react-native/test/index.tsx [hunk 1]
```diff
@@ -615,6 +615,7 @@ class WebViewTest extends React.Component {
             <WebView
                 originWhitelist={['https://origin.test']}
                 saveFormDataDisabled={false}
+                nativeConfig={{ component: 'test' }}
```
Comment (iRoachie) [inline types/react-native/test/index.tsx]: Could you add the other props here?

**Review by iRoachie — CHANGES_REQUESTED** [review]

**Comment by alloy** [comment]
It’s been red since https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/403199671 /cc @RyanCavanaugh

**Review by iRoachie — APPROVED** [review]

**Review by a-tarasyuk — APPROVED** [review]
LGTM

**Comment by RyanCavanaugh** [comment]
🌟 🎈 🎉 🏆 🎂 ✨ ⭐️ 

Congratulations on your first DefinitelyTyped contribution!

🌟 🎈 🎉 🏆 🎂 ✨ ⭐️

**Pull request merged**